src/common/utils/index.ts

```diff
@@ -72,11 +72,16 @@ export function formatInteger(num: number): string {
  *   formatDecimal(1.2345)    // 1.23
  *   formatDecimal(1.2345, 3) // 1.234
  */
-export const formatDecimal = (num: number, places = 2): string =>
-  num.toLocaleString(undefined, {
+export const formatDecimal = (num: number, places = 2): string => {
+  if (num === null) {
+    return '--';
+  }
```

Comment:
this is probably fine, but I'm mildly curious why it was necessary for this PR?

Reply:
There are a couple more null data points in some of the series that where causing some errors.  This was the best fix I came up with, but could put more effort into sussing out further up the chain. Also, I think other missing points might be an em-dash so i'll try to change that